src/renderer/views/Playlist/Playlist.js

```diff
@@ -59,20 +72,78 @@ export default defineComponent({
     },
     currentLocale: function () {
       return this.$i18n.locale.replace('_', '-')
-    }
+    },
+    playlistId: function() {
+      return this.$route.params.id
+    },
+    userPlaylistsReady: function () {
+      return this.$store.getters.getPlaylistsReady
+    },
+    selectedUserPlaylist: function () {
+      if (this.playlistId == null) { return null }
+      if (this.playlistId === '') { return null }
```

Comment:
Can be combined to a single `if`.
```suggestion
      if (this.playlistId == null || this.playlistId === '') { return null }
```